orte/mca/plm/rsh/plm_rsh_module.c

```diff
@@ -743,8 +779,10 @@ static void ssh_child(int argc, char **argv)
     close(fdin);
 
     /* close all file descriptors w/ exception of stdin/stdout/stderr */
-    for(fd=3; fd<fdmax; fd++)
-        close(fd);
+    if(close_open_file_descriptors() != ORTE_SUCCESS) {
+        for(fd=3; fd<fdmax; fd++)
```

Comment:
Minor style thing while you're editing this chunk of code: can you put the `close(fd)` in `{}`, too?  We typically enclose _all_ blocks in `{}`, even 1-line blocks.  Not sure how this one escaped... 😄